pyfr/integrators/dual/phys/steppers.py

```diff
@@ -27,11 +27,11 @@ def step(self, t, dt):
             self.pseudointegrator.init_stage(s, sc, dt)
             self.pseudointegrator.pseudo_advance(t + dt*tc)
 
-        self._finalize_step()
+        self._finalize_step(dt)
 
-    def _finalize_step(self):
+    def _finalize_step(self, dt):
         if not self.fsal:
-            bcoeffs = [bt*self._dt for bt in self.b]
+            bcoeffs = [bt*dt for bt in self.b]
```

Comment:
Why can `self.dt` not be used here?

Reply:
`self.dt` does not exist for `BaseDIRKStepper` and `self._dt` is as per the value given in the config file. We do have `dt = self.pseudointegrator.dt`.

I think we should rename `self._dt` --> `self._dt_base` to differentiate between what we are updating the dt to and what is given in the config file. We are no longer using self._dt anywhere outside the class. This could later be misleading.

I am setting `self.dt` as `self.pseudointegrator.dt` for now.